CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The public API of this library consists of the functions declared in file
 
 ## [Unreleased]
 ### Added
+- h3ToIjk function for getting IJK+ coordinates from an index (#83)
+- Internal `ijkDistance` function for determining the grid distance between IJK+ coordinates (#83)
+- Internal `h3Distance` function for determining the grid distance between H3 indexes (#83)
```

Comment:
Okay, you're right that it's not doing what I thought, but these CHANGELOG statements look wrong. `h3Distance` is marked as an export below, while `ijkDistance` and `h3ToIjk` are not.

Also the `h3ToIjk` filter application is not mentioned in these notes and I'm not entirely sure what it's purpose is, now?

Reply:
Oops, those are reversed. h3Distance should be the public one, the others are internal. I rebased and updated.

`h3ToIjk` is just for experimental use.